crates/omnix-cli/src/command/develop.rs

```diff
@@ -28,7 +28,7 @@ impl DevelopCommand {
     pub async fn run(&self) -> anyhow::Result<()> {
         let flake = self.flake_shell.without_attr();
 
-        let om_config = OmConfig::from_flake_url(NixCmd::get().await, &self.flake_shell).await?;
+        let om_config = OmConfig::from_flake_url(NixCmd::get().await, &flake).await?;
```

Comment:
I'm confused. This change impacts behaviour? `from_flake_url` already strips attr anyway. See 

![image](https://github.com/user-attachments/assets/b7c866e7-c606-4361-83db-15b43b236ab1)

and:

![image](https://github.com/user-attachments/assets/93bc0627-e343-464d-89ec-ccfa7750bb84)

Reply:
Oh I get it, the attr is expected to be config path, which is not always the case (as with nammayatri `.#backend`).